hermes/probe/create/create_file_test.go

```diff
@@ -0,0 +1,140 @@
+// TODO (#70) add license header and author line
+// TODO (#68) add doc strings
+
+package create
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/googleinterns/step224-2020/hermes/probe"
+	"github.com/googleinterns/step224-2020/hermes/probe/fakegcs"
+	"github.com/googleinterns/step224-2020/hermes/probe/metrics"
+
+	metricpb "github.com/google/cloudprober/metrics/proto"
+	probepb "github.com/googleinterns/step224-2020/config/proto"
+	journalpb "github.com/googleinterns/step224-2020/hermes/proto"
+)
+
+func TestFileName(t *testing.T) {
+	tests := []struct {
+		file     RandomFile
+		wantName string
+		wantErr  bool
+	}{
+		{RandomFile{51, 12}, "", true},
+		{RandomFile{0, 50}, "", true},
+		{RandomFile{3, 100}, "Hermes_03", false},
+		{RandomFile{12, 100}, "Hermes_12", false},
+		{RandomFile{3, 0}, "", true},
+		{RandomFile{3, 1001}, "", true},
+	}
+
+	for _, test := range tests {
+		got, err := test.file.FileName()
+		if err == nil {
+			if test.wantErr {
+				t.Errorf("{%v, %v}.FileName() failed expected an error got nil", test.file.ID, test.file.Size)
```

Comment:
nit: You should write this as if it was go code so `fmt.Errorf("test.file{%v}.FileName() = %v wanted %v, test.file, got, wantName`
You can do something similar for tests where you want an error (or don't want one, and expected one).

This should ideally mean you have less branches here.

Reply:
Will add that as a TODO changing error types and messages

Reply:
Added

Reply:
(#72 )

Reply:
changed the error messages!